src/syntax_highlight.rs

```diff
@@ -34,7 +36,8 @@ unsafe impl Sync for Setup {}
 lazy_static!{
     static ref SETUP: Setup = Setup {
         syntax_set: SyntaxSet::load_defaults_newlines(),
-        theme_set: ThemeSet::load_defaults()
+        theme_set: ThemeSet::load_defaults(),
+        theme_name: RefCell::new(DEFAULT_THEME_NAME.to_owned()),
```

Comment:
It doesn't give me a warm fuzzy feeling either.

It's just simple and works for this situation.  Should we want to add to it later to, I dunno, support varying syntax themes or anything that needs to change `theme_name` more than once, then it's worth overhauling the code to do away with this.

I might also be able to rewrite this with closures, but I fear implementation details in `syntax_highlight` will leak into the `document` module since Rust closures cannot be returned from a function, afaik.  I'll see what I come up with.

See my [other comment](https://github.com/cobalt-org/cobalt.rs/issues/202#issuecomment-313758943) regarding changing `liquid::Block` into a trait as one option.  I'd be happy to submit a PR for this.

Thanks for the review.